src/20_Components/amp-install-serviceworker.html

```diff
@@ -2,10 +2,12 @@
   #### Introduction
 
   The [amp-install-serviceworker](https://github.com/ampproject/amphtml/blob/master/extensions%2Famp-install-serviceworker%2Famp-install-serviceworker.md)
-  component enables Service Worker installation via original and cached AMP pages.
+  component enables service worker installation via original and cached AMP pages.
```

Comment:
Not sure if this is 100% correct right now--the iframe install is gated on a call to [`isProxyOrigin()`](https://github.com/ampproject/amphtml/blob/master/extensions/amp-install-serviceworker/0.1/amp-install-serviceworker.js#L45) which in turn is [hard-coded to `cdn.ampproject.org`](https://github.com/ampproject/amphtml/blob/master/src/url.js#L238). I haven't traced through exactly what happens, but there's definitely some differences in behavior depending on whether the document is on or off the cache. (Not just same versus cross-origin.)